### ISSUE TITLE
Documentation Missing?

### ISSUE THREAD

**Issue by PhilETaylor** [phpseclib/phpseclib#1965]
The https://phpseclib.com/ just shows this 

![ScreenShot-2023-11-29-16 08 51](https://github.com/phpseclib/phpseclib/assets/400092/9d91d2b1-7e98-43b9-9f75-dd42fe7a9f7a)

**Comment by terrafrost** [comment]
github broke it I'd say. A misspelling in the documentation [was fixed](https://github.com/phpseclib/phpseclib.github.io/pull/1). I merged the PR, which triggered GitHub Actions's CI/CD pipeline to rebuild the documentation but because the `actions/setup-node@v3` module that .github/workflows/ci.yml uses had a BC breaking change the documentation was unable to build successfully.

Here was the fix:

https://github.com/phpseclib/phpseclib.github.io/commit/3223836bf48b3805030d4b6afe7363cccde2bf96

That was a bit of a PITA to figure out. Had to remove the timestamps from the logs, then do a visual diff of the logs from a successful run from a month ago and this most recent one. Based on the differences I saw I thought that the above commit might do the trick and it did.
